armi/reactor/converters/axialExpansionChanger.py

```diff
@@ -192,7 +193,7 @@ def setAssembly(self, a, setFuel=True):
             This is useful when target components within a fuel block need to be determined on-the-fly.
         """
         self.linked = AssemblyAxialLinkage(a)
-        self.expansionData = ExpansionData(a, setFuel)
+        self.expansionData = ExpansionData(a, setFuel, coldHeightsToHot)
```

Comment:
prefer when calling to pass kwargs directly

`self.expansionData = ExpansionData(a, setFuel=setFuel, coldHeightsToHot=coldHeightsToHot)`

in this case if kwargs are re-ordered or one is removed you are still safe

Reply:
https://github.com/terrapower/armi/pull/1342/commits/73c16ed9d0a14ffbf3090e619426e6c740fc38b4